core/src/main/java/com/valtech/aemsaas/core/util/Constants.java

```diff
@@ -0,0 +1,10 @@
+package com.valtech.aemsaas.core.util;
+
+public interface Constants {
```

Comment:
We shouldn't use this constant class anymore. Instead add those to the relevant class